Dockerfile

```
@@ -0,0 +1,4 @@
+FROM node:12
```

```suggestion
FROM node:12-alpine
```
I think it is save to use the smaller alpine bistro here, what do you think?

---

I had tried that initially - didn't work. Tried it again, turns out it was just missing some dependencies to run the binary, which I added to this Dockerfile :) Thanks for the review